examples/nomad-consul-ami/nomad-consul.json

```diff
@@ -2,9 +2,9 @@
   "min_packer_version": "0.12.0",
   "variables": {
     "aws_region": "us-east-1",
-    "nomad_version": "0.7.1",
-    "consul_module_version": "v0.3.1",
-    "consul_version": "1.0.3"
+    "nomad_version": "0.8.6",
+    "consul_module_version": "master",
```

Comment:
We probably should use a specific tag rather than master, as we occasionally have to make backwards incompatible changes (e.g., Terraform 0.12 is coming!).